FirebaseAuth/Sources/AuthProvider/OAuth/FIROAuthProvider.m

```diff
@@ -345,6 +353,28 @@ - (void)getHeadFulLiteURLWithEventID:(NSString *)eventID
                                        urlArguments[@"hl"] =
                                            strongSelf->_auth.requestConfiguration.languageCode;
                                      }
+
+                                     if (appCheck) {
+                                       [appCheck
+                                           getTokenForcingRefresh:false
+                                                       completion:^(
+                                                           id<FIRAppCheckTokenResultInterop> _Nonnull tokenResult) {
+                                                         if (tokenResult.error) {
+                                                           FIRLogWarning(
+                                                               kFIRLoggerAuth, @"I-AUT000018",
+                                                               @"Error getting App Check token; "
+                                                               @"using placeholder token "
+                                                               @"instead. Error: %@",
+                                                               tokenResult.error);
+                                                         }
+                                                         NSString *appCheckTokenFragments =
+                                                             [kAppCheckTokenURLFormat
+                                                                 stringByAppendingString:
+                                                                     tokenResult.token];
+                                                         urlArguments[@"appCheckToken"] =
```

Comment:
One thing to note -- the App Check token is being attached as a URL fragment (i.e. `#fac=app.check.token`, not a query param, so this shouldn't go in the `urlArguments` dictionary.